src/components/SideBar/SideBarComponents/FacetFilters.js

```diff
@@ -415,229 +346,90 @@ const FacetPanel = ({ classes }) => {
-                          { sideBarItem.slider !== true
-                            && (
-                              <div
-                                className={classes.sortGroup}
-                              >
-                                <span
-                                  className={classes.sortGroupIcon}
-                                >
-                                  <Icon
-                                    onClick={handleGroupReset(
-                                      sideBarItem,
-                                      sideBarIndex,
-                                    )}
-                                    style={{ fontSize: 15 }}
-                                  >
-                                    <img
-                                      src={resetIconFilter.src}
-                                      height={size}
-                                      width={size}
-                                      alt={resetIconFilter.alt}
-                                    />
-                                  </Icon>
-                                </span>
-                                <span
-                                  className={classes.sortGroupItem}
-                                  style={{ color: getSortButtonColor(sideBarItem, 'alphabet') }}
-                                  onClick={() => {
-                                    sortSection(sideBarItem.groupName, 'alphabet');
-                                  }}
-                                >
-                                  {sortLabels.sortAlphabetically}
-                                </span>
-                                <span
-                                  className={classes.sortGroupItemCounts}
-                                  style={{ color: getSortButtonColor(sideBarItem, 'count') }}
-                                  onClick={() => {
-                                    sortSection(sideBarItem.groupName, 'count');
-                                  }}
-                                >
-                                  {sortLabels.sortByCount}
-                                </span>
-                              </div>
-                            )}
-                          {sideBarItem.slider === true
-                            && (
-                              <div
-                                className={
-                                  classes.sortGroup
-                                }
+                          <div
+                            className={classes.sortGroup}
+                          >
+                            <span
+                              className={classes.sortGroupIcon}
+                            >
+                              <Icon
+                                onClick={handleGroupReset(
+                                  sideBarItem.datafield, sideBarItem.groupName,
+                                )}
```

Comment:
Codacy has a fix for the issue: [Unexpected newline before ')'.](https://app.codacy.com/gh/CBIIT/bento-frontend/pullRequest?prid=8502143)
```suggestion
                                style={{ fontSize: 15 }}
```